src/api/annotation/AnnotationAPI.js

```diff
@@ -139,16 +166,18 @@ export default class AnnotationAPI extends EventEmitter {
         const success = await this.openmct.objects.save(createdObject);
         if (success) {
             this.emit('annotationCreated', createdObject);
-            this.#updateAnnotationModified(domainObject);
+            Object.values(targetDomainObjects).forEach(targetDomainObject => {
+                this.#updateAnnotationModified(targetDomainObject);
+            });
 
             return createdObject;
         } else {
             throw new Error('Failed to create object');
         }
     }
 
-    #updateAnnotationModified(domainObject) {
-        this.openmct.objects.mutate(domainObject, this.ANNOTATION_LAST_CREATED, Date.now());
+    #updateAnnotationModified(targetDomainObject) {
+        this.openmct.objects.mutate(targetDomainObject, this.ANNOTATION_LAST_CREATED, Date.now());
```

Comment:
I'll defer to @akhenry, but I don't think it matters. This is mainly for keeping track of when to check for new annotations coming across the wire.

Reply:
It's a good question, I think the answer is that `Date.now()` is ok in this case, for the same reason we use it for modified and created timestamps for objects. We also don't have a ubiquitous clock that can be queried for simulated time in Open MCT...yet. Right now it can only be queried in real-time mode.